alibi/explainers/backends/pytorch/cfrl_base.py

```diff
@@ -63,9 +63,9 @@ def __init__(self,
                                         batch_size=self.batch_size)
 
         # Define number of classes for classification & minimum and maximum labels for regression
-        self.num_classes: Optional[int] = None
-        self.min_m: Optional[float] = None
-        self.max_m: Optional[float] = None
+        self.num_classes: int
+        self.min_m: float
+        self.max_m: float
```

Comment:
I avoided initializing these to `None` as those throw `mypy` errors when passed into the `numpy.random.randint` function.